ci-operator/step-registry/cucushift/installer/check/ibmcloud/resourcegroup/cucushift-installer-check-ibmcloud-resourcegroup-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: cucushift-installer-check-ibmcloud-resourcegroup
+  from_image:
+    namespace: ocp
+    name: "4.13"
+    tag: upi-installer
+  grace_period: 10m
+  commands: cucushift-installer-check-ibmcloud-resourcegroup-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: CREATE_CLUSTER_RESOURCE_GROUP 
```

Comment:
Per `ibmcloud-provision-resourcegroup`, its default value is "no", so here it should be "no" to keep consistent